JapaneseAddress/Block/Checkout/BillingAddressComponent.php

```diff
@@ -16,11 +18,36 @@
  */
 class BillingAddressComponent implements LayoutProcessorInterface
 {
+    /**
+     * @var ScopeConfigInterface
+     */
+    private $scopeConfig;
+
+    /**
```

Comment:
Looks like empty block is redundant.

Reply:
removed this and use already defined constant.